docs/faqs.md

```diff
@@ -29,11 +29,13 @@ to your own `.env` file.
 ## How do I see comments in the Feed?
 
 On the home Feed, we only show comments above certain "score". It's likely the
-comments in the local environment will never meet this score. If you want to see comments locally, you will need to
-update the score of your local comments manually. Here's how:
+comments in the local environment will never meet this score. If you want to see
+comments locally, you will need to update the score of your local comments
+manually. Here's how:
 
 1. Open the terminal.
-2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL terminal.
+2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
```

Comment:
**suggestion (non-blocking):** Unrelated, but at one point we could change this to also mention `rails dbconsole`, which parses `config/database.yml`.

Reply:
TIL 👏🏻